[2025-06-12 20:56:05] [31merror[39m: MongooseError: Connection.prototype.close() no longer accepts a callback
    at NativeConnection.close (D:\test project 3\online-literacy-platform\backend\node_modules\mongoose\lib\connection.js:1205:11)
    at Server.<anonymous> (D:\test project 3\online-literacy-platform\backend\server.js:25:27)
    at Object.onceWrapper (node:events:638:28)
    at Server.emit (node:events:524:28)
    at emitCloseNT (node:net:2344:8)
    at process.processTicksAndRejections (node:internal/process/task_queues:81:21)
[2025-06-12 20:56:55] [31merror[39m: MongooseError: Connection.prototype.close() no longer accepts a callback
    at NativeConnection.close (D:\test project 3\online-literacy-platform\backend\node_modules\mongoose\lib\connection.js:1205:11)
    at Server.<anonymous> (D:\test project 3\online-literacy-platform\backend\server.js:25:27)
    at Object.onceWrapper (node:events:638:28)
    at Server.emit (node:events:524:28)
    at emitCloseNT (node:net:2344:8)
    at process.processTicksAndRejections (node:internal/process/task_queues:81:21)
[2025-06-12 20:57:26] [31merror[39m: MongooseError: Connection.prototype.close() no longer accepts a callback
    at NativeConnection.close (D:\test project 3\online-literacy-platform\backend\node_modules\mongoose\lib\connection.js:1205:11)
    at Server.<anonymous> (D:\test project 3\online-literacy-platform\backend\server.js:25:27)
    at Object.onceWrapper (node:events:638:28)
    at Server.emit (node:events:524:28)
    at emitCloseNT (node:net:2344:8)
    at process.processTicksAndRejections (node:internal/process/task_queues:81:21)
[2025-06-12 20:58:00] [31merror[39m: MongooseError: Connection.prototype.close() no longer accepts a callback
    at NativeConnection.close (D:\test project 3\online-literacy-platform\backend\node_modules\mongoose\lib\connection.js:1205:11)
    at Server.<anonymous> (D:\test project 3\online-literacy-platform\backend\server.js:25:27)
    at Object.onceWrapper (node:events:638:28)
    at Server.emit (node:events:524:28)
    at emitCloseNT (node:net:2344:8)
    at process.processTicksAndRejections (node:internal/process/task_queues:81:21)
[2025-06-12 20:58:59] [31merror[39m: MongooseError: Connection.prototype.close() no longer accepts a callback
    at NativeConnection.close (D:\test project 3\online-literacy-platform\backend\node_modules\mongoose\lib\connection.js:1205:11)
    at Server.<anonymous> (D:\test project 3\online-literacy-platform\backend\server.js:25:27)
    at Object.onceWrapper (node:events:638:28)
    at Server.emit (node:events:524:28)
    at emitCloseNT (node:net:2344:8)
    at process.processTicksAndRejections (node:internal/process/task_queues:81:21)
[2025-06-12 20:59:23] [31merror[39m: MongooseError: Connection.prototype.close() no longer accepts a callback
    at NativeConnection.close (D:\test project 3\online-literacy-platform\backend\node_modules\mongoose\lib\connection.js:1205:11)
    at Server.<anonymous> (D:\test project 3\online-literacy-platform\backend\server.js:25:27)
    at Object.onceWrapper (node:events:638:28)
    at Server.emit (node:events:524:28)
    at emitCloseNT (node:net:2344:8)
    at process.processTicksAndRejections (node:internal/process/task_queues:81:21)
[2025-06-12 20:59:59] [31merror[39m: MongooseError: Connection.prototype.close() no longer accepts a callback
    at NativeConnection.close (D:\test project 3\online-literacy-platform\backend\node_modules\mongoose\lib\connection.js:1205:11)
    at Server.<anonymous> (D:\test project 3\online-literacy-platform\backend\server.js:25:27)
    at Object.onceWrapper (node:events:638:28)
    at Server.emit (node:events:524:28)
    at emitCloseNT (node:net:2344:8)
    at process.processTicksAndRejections (node:internal/process/task_queues:81:21)
[2025-06-12 21:00:22] [31merror[39m: MongooseError: Connection.prototype.close() no longer accepts a callback
    at NativeConnection.close (D:\test project 3\online-literacy-platform\backend\node_modules\mongoose\lib\connection.js:1205:11)
    at Server.<anonymous> (D:\test project 3\online-literacy-platform\backend\server.js:25:27)
    at Object.onceWrapper (node:events:638:28)
    at Server.emit (node:events:524:28)
    at emitCloseNT (node:net:2344:8)
    at process.processTicksAndRejections (node:internal/process/task_queues:81:21)
[2025-06-12 21:00:34] [31merror[39m: MongooseError: Connection.prototype.close() no longer accepts a callback
    at NativeConnection.close (D:\test project 3\online-literacy-platform\backend\node_modules\mongoose\lib\connection.js:1205:11)
    at Server.<anonymous> (D:\test project 3\online-literacy-platform\backend\server.js:25:27)
    at Object.onceWrapper (node:events:638:28)
    at Server.emit (node:events:524:28)
    at emitCloseNT (node:net:2344:8)
    at process.processTicksAndRejections (node:internal/process/task_queues:81:21)
[2025-06-12 21:01:01] [31merror[39m: MongooseError: Connection.prototype.close() no longer accepts a callback
    at NativeConnection.close (D:\test project 3\online-literacy-platform\backend\node_modules\mongoose\lib\connection.js:1205:11)
    at Server.<anonymous> (D:\test project 3\online-literacy-platform\backend\server.js:25:27)
    at Object.onceWrapper (node:events:638:28)
    at Server.emit (node:events:524:28)
    at emitCloseNT (node:net:2344:8)
    at process.processTicksAndRejections (node:internal/process/task_queues:81:21)
[2025-06-12 21:01:33] [31merror[39m: MongooseError: Connection.prototype.close() no longer accepts a callback
    at NativeConnection.close (D:\test project 3\online-literacy-platform\backend\node_modules\mongoose\lib\connection.js:1205:11)
    at Server.<anonymous> (D:\test project 3\online-literacy-platform\backend\server.js:25:27)
    at Object.onceWrapper (node:events:638:28)
    at Server.emit (node:events:524:28)
    at emitCloseNT (node:net:2344:8)
    at process.processTicksAndRejections (node:internal/process/task_queues:81:21)
[2025-06-12 21:01:47] [31merror[39m: MongooseError: Connection.prototype.close() no longer accepts a callback
    at NativeConnection.close (D:\test project 3\online-literacy-platform\backend\node_modules\mongoose\lib\connection.js:1205:11)
    at Server.<anonymous> (D:\test project 3\online-literacy-platform\backend\server.js:25:27)
    at Object.onceWrapper (node:events:638:28)
    at Server.emit (node:events:524:28)
    at emitCloseNT (node:net:2344:8)
    at process.processTicksAndRejections (node:internal/process/task_queues:81:21)
[2025-06-12 21:01:59] [31merror[39m: MongooseError: Connection.prototype.close() no longer accepts a callback
    at NativeConnection.close (D:\test project 3\online-literacy-platform\backend\node_modules\mongoose\lib\connection.js:1205:11)
    at Server.<anonymous> (D:\test project 3\online-literacy-platform\backend\server.js:25:27)
    at Object.onceWrapper (node:events:638:28)
    at Server.emit (node:events:524:28)
    at emitCloseNT (node:net:2344:8)
    at process.processTicksAndRejections (node:internal/process/task_queues:81:21)
[2025-06-12 21:02:15] [31merror[39m: MongooseError: Connection.prototype.close() no longer accepts a callback
    at NativeConnection.close (D:\test project 3\online-literacy-platform\backend\node_modules\mongoose\lib\connection.js:1205:11)
    at Server.<anonymous> (D:\test project 3\online-literacy-platform\backend\server.js:25:27)
    at Object.onceWrapper (node:events:638:28)
    at Server.emit (node:events:524:28)
    at emitCloseNT (node:net:2344:8)
    at process.processTicksAndRejections (node:internal/process/task_queues:81:21)
[2025-06-12 21:02:28] [31merror[39m: MongooseError: Connection.prototype.close() no longer accepts a callback
    at NativeConnection.close (D:\test project 3\online-literacy-platform\backend\node_modules\mongoose\lib\connection.js:1205:11)
    at Server.<anonymous> (D:\test project 3\online-literacy-platform\backend\server.js:25:27)
    at Object.onceWrapper (node:events:638:28)
    at Server.emit (node:events:524:28)
    at emitCloseNT (node:net:2344:8)
    at process.processTicksAndRejections (node:internal/process/task_queues:81:21)
[2025-06-12 21:05:52] [31merror[39m: MongooseError: Connection.prototype.close() no longer accepts a callback
    at NativeConnection.close (D:\test project 3\online-literacy-platform\backend\node_modules\mongoose\lib\connection.js:1205:11)
    at Server.<anonymous> (D:\test project 3\online-literacy-platform\backend\server.js:25:27)
    at Object.onceWrapper (node:events:638:28)
    at Server.emit (node:events:524:28)
    at emitCloseNT (node:net:2344:8)
    at process.processTicksAndRejections (node:internal/process/task_queues:81:21)
